### PR TITLE
Example Dataset - Notebook 4

### PR DESCRIPTION
**Pull request by srivarra**
**If you haven't already, please read through our [contributing guidelines](https://ark-analysis.readthedocs.io/en/latest/_rtd/contributing.html) before opening your PR**

**What is the purpose of this PR?**

Addresses Part 3 of #692. Adds downloading the minimum datasets required for Notebook 4.

**How did you implement your changes**

Updates the process for Notebook 4. Updated the dataset on Hugging Face as well.

**Remaining issues**

None ATM.
* [x] Need to run through the notebooks on the Docker to make sure everything is working properly.

### CODE REVIEW

**Review by ngreenwald — CHANGES_REQUESTED** [review]
What's different about the notebook 3 and notebook 4 datasets?

**Comment by srivarra** [comment]
@ngreenwald The notebook 4 dataset contains `example_cell_output`, the data generated by notebook 3 which is needed for post clustering.

**Comment by srivarra** [comment]
@ngreenwald Are you referencing the step when the metacluster labels get added to the table? If so those are saved in a separate file which is suffixed by `_cell_labels` (i.e. `cell_table_size_normalized_cell_labels.csv`)

**Comment by ngreenwald** [comment]
Got it, sounds good

**Review by ngreenwald — CHANGES_REQUESTED** [review]
Same merge conflicts here as well

File: templates/4_Post_Clustering.ipynb [hunk 1]
```diff
@@ -24,7 +24,8 @@
     "import numpy as np\n",
```
Comment (ngreenwald) [inline templates/4_Post_Clustering.ipynb:24]: <p>This cell needs to be updated. The goal is to show how to update the cell table using the threshold identified in the histogram above. This is still using ECAD</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/763/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

**Review by ngreenwald — CHANGES_REQUESTED** [review]
Looks good, just need to update the notebook comments

File: templates/4_Post_Clustering.ipynb (hunk 1 above)
Comment (ngreenwald) [inline templates/4_Post_Clustering.ipynb:24]: <p>Will you add a link to this notebook (and the others) that import the example dataset with a link to the readme section, saying that there's more information there</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/763/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (ngreenwald) [inline templates/4_Post_Clustering.ipynb:24]: <p>Looks like not quite all of the changes we discussed last week got included. </p><p>The code above is identifying cells in the immune_other category which express CD3 as strongly as CD4T and CD8Ts do. These are both Tcell populations, so the cells from immune_other which have high levels of CD3 should be assigned to a new category, tcell_other. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/763/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

**Review by ngreenwald — CHANGES_REQUESTED** [review]
Notebook changes look great, looks like there's a travis issue

**Comment by srivarra** [comment]
@ngreenwald Yeah, it's something on test PYPIs end it seems.

**Review by ngreenwald — APPROVED** [review]
Okay, the actual build is passing, I'll go ahead and merge this in. Will you run all four notebooks after clearing your cache to make sure there aren't any issues that we missed with the download logic?

**Pull request merged**